gitevanhsu/book-reviews
# Feature/profile layout

**Description**
1. Completed profile page and other member page with RWD.
2. Completed mobile side profile bookshelf modify with button.
3. Completed sign in/up page with RWD.
4. Completed friend system.

**Test plan**
1. Try use bookshelf button to move book to other bookshelf in mobile page.
2. Try sent a friend request to other member, check state change on different state.
